packages/rancher-istio/charts/values.yaml

```
@@ -61,18 +77,15 @@ meshConfig:
 kiali:
   enabled: true
   auth:
-    # todo: what auth strategy ?
-    # strategy: token # this is default, you can use a service-account or kubeconfig token
     strategy: anonymous
   deployment:
+    ##TODO: update kiali to allow for repository and image
     ingress_enabled: false
+    image_name: docker.io/rancher/kiali-kiali
+    image_version: v1.22.1
   external_services:
     prometheus:
       custom_metrics_url: "http://rancher-monitoring-prometheus.monitoring-system.svc:9090"
       url: "http://rancher-monitoring-prometheus.monitoring-system.svc:9090"
```

I don't think I caught this in a previous PR, but these URLs need to be updated based on the recent `rancher-monitoring` namespace changes:
```suggestion
      custom_metrics_url: "http://rancher-monitoring-prometheus.cattle-monitoring-system.svc:9090"
      url: "http://rancher-monitoring-prometheus.cattle-monitoring-system.svc:9090"
```
Not sure if this can be addressed in this PR or if we want to address this in a separate PR. It also might require changes to the `README.md` iirc for Istio.

---

I went ahead and added it to this pr, we currently have it in the readme as ```http://{{ .Values.nameOverride }}-prometheus.{{ .Values.namespaceOverride }}.svc:{{ prometheus.service.port }}``` so I thiiink we should be okay